libraries/exporters/src/main/java/com/siemens/sw360/exporter/ComponentExporter.java

```diff
@@ -9,41 +9,80 @@
-import java.util.List;
+import java.util.*;
+import java.util.stream.Collectors;
 
 import static com.google.common.base.Strings.nullToEmpty;
-import static com.siemens.sw360.datahandler.common.CommonUtils.joinStrings;
+import static com.siemens.sw360.datahandler.common.CommonUtils.nullToEmptySet;
+import static com.siemens.sw360.datahandler.common.SW360Utils.fieldValueAsString;
+import static com.siemens.sw360.datahandler.common.SW360Utils.getReleaseNames;
+import static com.siemens.sw360.datahandler.thrift.components.Component._Fields.*;
 
-/**
- * Created by bodet on 06/02/15.
- *
- * @author cedric.bodet@tngtech.com
- */
 public class ComponentExporter extends ExcelExporter<Component> {
+    private static final Logger log = Logger.getLogger(ProjectExporter.class);
+    private static boolean extendedByReleases;
+    private static ReleaseHelper releaseHelper;
+
+    public static final Map<String, String> nameToDisplayName;
+    static {
+        nameToDisplayName = new HashMap<>();
+        nameToDisplayName.put(Component._Fields.NAME.getFieldName(), "component name");
+        nameToDisplayName.put(Component._Fields.CREATED_ON.getFieldName(), "creation date");
+        nameToDisplayName.put(Component._Fields.COMPONENT_TYPE.getFieldName(), "component type");
+        nameToDisplayName.put(Component._Fields.CREATED_BY.getFieldName(), "created by");
+        nameToDisplayName.put(Component._Fields.RELEASE_IDS.getFieldName(), "releases");
+        nameToDisplayName.put(Component._Fields.MAIN_LICENSE_IDS.getFieldName(), "main license IDs");
+        nameToDisplayName.put(Component._Fields.SOFTWARE_PLATFORMS.getFieldName(), "software platforms");
+        nameToDisplayName.put(Component._Fields.OPERATING_SYSTEMS.getFieldName(), "operating systems");
+        nameToDisplayName.put(Component._Fields.VENDOR_NAMES.getFieldName(), "vendor names");
+    }
 
-    private static final List<String> HEADERS = ImmutableList.<String>builder()
-            .add("Component Name")
-            .add("Programming Languages")
-            .add("Categories")
-            .add("Operating Systems")
-            .add("Software Platforms")
-            .add("Created by")
-            .add("Creation Date")
-            .add("Releases")
+    private static final List<Component._Fields> IGNORED_FIELDS = ImmutableList.<Component._Fields>builder()
+            .add(ID)
+            .add(REVISION)
+            .add(DOCUMENT_STATE)
+            .add(PERMISSIONS)
+            .add(RELEASES)
             .build();
 
-    public ComponentExporter() {
-        super(new ComponentHelper());
+    public static final List<Component._Fields> RENDERED_FIELDS = Component.metaDataMap.keySet()
+            .stream()
+            .filter(k -> ! IGNORED_FIELDS.contains(k))
+            .collect(Collectors.toList());
+
+    protected static List<String> HEADERS = new ArrayList<>();
+
+    public ComponentExporter(ComponentService.Iface componentClient, boolean extendedByReleases) {
```

Comment:
In the call of this function you name the boolean variable `isExtendedExcelExport`. It would be nice to have more consistency here.